src/ARCExpect/ErrorMessage.fs

```diff
@@ -85,6 +85,45 @@ type ErrorMessage =
             str.AppendFormat(" > line '{0}'", line) |> ignore
         | None -> ()
 
+        match Param.tryGetValueOfCvParamAttr "Position" iParam with
+        | Some position -> 
+            str.AppendFormat(" > position '{0}'", position) |> ignore
+        | None -> ()
+        str.ToString()
+
+
+    static member ofIParamCollection error iParamCollection =
+
+        let iParam = Seq.head iParamCollection
```

Comment:
I do not get these functions to be honest. The API naming indicates that it generates error messages from `IParam` collections, but you only use the first item in the sequence to generate the error message

Reply:
Yes, since printing all items would be too much. I mean, this is arbitrary. How many items do we want to display? I chose 1, but we could also do 3 or 5 or any number...
What's your opinion on this?

Reply:
> I mean, this is arbitrary.

Not really, ideally you want to know all items that do not satisfy the predicate